docs/releases/1.11.txt

```diff
@@ -533,3 +533,8 @@ Miscellaneous
   :ttag:`{% include %} <include>` template tag is deprecated as the behavior is
   often more confusing than helpful. In Django 2.1, the exception will be
   raised.
+
+* ``DatabaseIntrospection.get_indexes()`` is deprecated in favor of
+  ``DatabaseIntrospection.get_constraints()``. The :djadmin:`inspectdb` command
```

Comment:
Please move the second sentence to "Database backend API" under backwards-incompatible changes, e.g. :djadmin:`inspectdb` no longer calls DatabaseIntrospection.get_indexes(). Add unique indexes to the return value of get_constraints().